README.md

```diff
@@ -39,19 +39,14 @@ With the ESnext field declarations proposal, the above example can be written as
 class Counter extends HTMLElement {
```

Comment:
@ljharb what would be the functional difference between a shadowing bound function and an instance arrow function, if any?